plugin.video.powerunlimited/addon.xml

```diff
@@ -2,13 +2,15 @@
 <addon 
 	id="plugin.video.powerunlimited" 
 	name="PowerUnlimited Tv" 
-	version="1.0.6"
+	version="1.0.7"
 	provider-name="Skipmode A1">
   <requires>
-    <import addon="xbmc.python"                 version="2.14.0"/>
-	<import addon="script.module.beautifulsoup" version="3.0.8"/>
-	<import addon="plugin.video.youtube" 		version="5.1.7" />
- 	<import addon="script.module.requests"      version="2.4.3"/>
+    <import addon="xbmc.python"                     version="2.14.0"/>
+    <import addon="script.module.beautifulsoup4"    version="4.5.3"/>
```

Comment:
@skipmodea1 ERROR: Version mismatch for addon script.module.beautifulsoup4. Required: 4.5.3, Available: 4.3.2